docs/zh/latest/architecture-design/apisix.md

```diff
@@ -25,6 +25,14 @@ title: APISIX
 
 ![软件架构](../../../assets/images/flow-software-architecture.png)
 
+Apache APISIX 是一个动态、实时、高性能的云原生 API 网关。它构建于 Nginx + ngx_lua 的技术基础之上，充分利用了 luajit 所提供的强大性能。
+
+APISIX 主要分为两个部分，其一是 APISIX 核心，包括 Lua 插件、多语言插件运行时、WASM 插件运行时等；其二是功能丰富的各种内置插件，包括可观测性、安全、流量控制等。
```

Comment:
```suggestion
APISIX 主要分为两个部分，其一是 APISIX 核心功能，包括 Lua 插件、多语言插件运行时、Wasm 插件运行时等；其二是功能丰富的各种内置插件，包括可观测性、安全、流量控制等。
```

Reply:
This corresponds to the `APISIX Core` in the architecture diagram.
![image](https://user-images.githubusercontent.com/8078418/177235307-34bc99da-f9aa-4d3c-a746-57bcfcf4758d.png)